crates/cast/tests/cli/main.rs

```diff
@@ -1295,6 +1295,48 @@ casttest!(storage_layout_complex, |_prj, cmd| {
 ╰-------------------------------+--------------------------------------------------------------------+------+--------+-------+--------------------------------------------------+--------------------------------------------------------------------+---------------------------------╯
 
 
+"#]]);
+});
+
+casttest!(storage_layout_complex_proxy, |_prj, cmd| {
+    cmd.args([
+        "storage",
+        "--rpc-url",
+        next_rpc_endpoint(NamedChain::Sepolia).as_str(),
+        "--block",
+        "7857852",
+        "--etherscan-api-key",
+        next_mainnet_etherscan_api_key().as_str(),
+        "0xE2588A9CAb7Ea877206E35f615a39f84a64A7A3b",
+        "--proxy",
+        "0x29fcb43b46531bca003ddc8fcb67ffe91900c762"
+    ])
+    .assert_success()
+    .stdout_eq(str![[r#"
```

Comment:
I think this will be a pain to update in case anything changes with the layout

@grandizzy should be put this in a file instead and use include_str! or does that not work with the str! macro

Reply:
I think that's fine, we can easily update by rerunning the test with ` SNAPSHOTS=overwrite` in case it gets changed